classes/SectionView/SeneddView.php

```diff
@@ -122,11 +118,7 @@ protected function getSearchBox(array $data): \MySociety\TheyWorkForYou\Search\S
                 $constituencies[$member["constituency"]] = 1;
             }
             $constituencies = array_keys($constituencies);
-            if (LANGUAGE == 'cy'){
-                $search_box->add_quick_link('Darganfod mwy am eich Aelodau Cynulliad dros ' . $constituencies[0] . ' a ' . $constituencies[1], '/postcode/?pc=' . $data["mp_data"]['postcode'],'torso');
-            } else {
-                $search_box->add_quick_link('Find out more about your MSs for ' . $constituencies[0] . ' and ' . $constituencies[1], '/postcode/?pc=' . $data["mp_data"]['postcode'],'torso');
-            }
+        $search_box->add_quick_link(gettext('Find out more about your MSs for ') . $constituencies[0] . gettext(' and ') . $constituencies[1], '/postcode/?pc=' . $data["mp_data"]['postcode'],'torso');
```

Comment:
Sorry, I realise I asked for this, but good knowledge to share - unless you know the translated language has the same construction as English (I don't know if Welsh does here), this should be `sprintf(gettext("Find out more about your MSs for %s and %s"), $constituencies[0], $constituencies[1])` so that the sentence can reorder if the language requires that.

Reply:
Yes that seemed the wrong way (fine for welsh, but good to learn!)